### PR TITLE
Remove the word "successfully"

### PR DESCRIPTION
**Pull request by weilirs**
Signed-off-by: weilirs <weilirs@umich.edu>

**Notes for Reviewers**
In the last week's development call, Lee told me to remove the word "successfully" from Meshery events, error code messages, and notifications sent to the user.
- This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/internal/cli/root/design/error.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func ErrDesignNotFound() error {
 }
 
 func ErrInvalidDesignFile(err error) error {
-	return errors.New(ErrInvalidDesignFileCode, errors.Fatal, []string{err.Error()}, []string{"Design appears invalid. Could not parse successfully"}, []string{"Design file provided is not valid"}, []string{"Please check that your design file is a valid yaml file"})
+	return errors.New(ErrInvalidDesignFileCode, errors.Fatal, []string{err.Error()}, []string{"Design appears invalid. Could not parse"}, []string{"Design file provided is not valid"}, []string{"Please check that your design file is a valid yaml file"})
```
Comment (alphaX86) [inline mesheryctl/internal/cli/root/design/error.go:56]:
```suggestion
	return errors.New(ErrInvalidDesignFileCode, errors.Fatal, []string{err.Error()}, []string{"Design appears invalid. Could not parse provided design"}, []string{"Design file provided is not valid"}, []string{"Please check that your design file is a valid yaml file"})
```

File: mesheryctl/pkg/utils/auth.go [hunk 2]
```diff
@@ -332,7 +332,7 @@ func initiateRemoteProviderAuth(provider Provider) (string, error) {
 			return
 		}
 
-		fmt.Fprint(rw, "successfully logged in, you can close this window now")
+		fmt.Fprint(rw, "logged in, you can close this window now")
```
Comment (alphaX86) [inline mesheryctl/pkg/utils/auth.go:335]:
```suggestion
		fmt.Fprint(rw, "you have logged in, you can close this window now")
```

**Review by alphaX86 — CHANGES_REQUESTED** [review]

**Comment by vishalvivekm** [comment]
@weilirs 
Thank you for your contributions! Let's discuss this today, during Meshery Development call at 9:00 AM CT | 8:30 PM IST. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1&tab=t.0#) || [meeting link.](https://meet.layer5.io/meshery)

**Comment by weilirs** [comment]
> @weilirs Thank you for your contributions! Let's discuss this today, during Meshery Development call at 9:00 AM CT | 8:30 PM IST. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1&tab=t.0#) || [meeting link.](https://meet.layer5.io/meshery)

sorry I can't make it today

**Review by alphaX86 — DISMISSED** [review]
Approved on my side. @weilirs pls fix merge conflicts.

@lekaf974 could you review this once?

**Comment by weilirs** [comment]
> Approved on my side. @weilirs pls fix merge conflicts.
> 
> @lekaf974 could you review this once?

I fixed the merge conflict, I don't know why it dismissed your review...

**Review by alphaX86 — APPROVED** [review]

**Comment by alphaX86** [comment]
> > Approved on my side. @weilirs pls fix merge conflicts.
> > 
> > @lekaf974 could you review this once?
> 
> I fixed the merge conflict, I don't know why it dismissed your review...

Not a problem @weilirs :) no worries

**Review by leecalcote — APPROVED** [review]
Awesome. Thank you, @weilirs and @alphaX86. 👏

**Pull request merged**